modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -535,6 +541,11 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		_commercePriceEntryLocalService.addCommercePriceEntry(externalReferenceCode, cProductId, cpInstanceUuid, commercePriceListId, price, false, promoPrice, null, serviceContext);
 	}
 
+	public void method(String className, long classPK, ThemeDisplay themeDisplay) {
```

Comment:
Good job @kyle-miho! It was an edge case. Be careful when changing the CatchAll logic, especially with its regex, as it deals with different automation patterns. But I liked the solution and as long as the test continues to pass, I don't think any other cases will have been impacted.

Also, to reinforce this on the PR validation as well, before we create a test suite case, especially for Upgrade SF checks. It will run all upgrade tests. This way we test not only the specific check but all others as well.

ci:test:hotel